packages/create-sitecore-jss/src/initializers/vue/index.ts

```diff
@@ -0,0 +1,50 @@
+﻿import path from 'path';
+import { prompt } from 'inquirer';
+import { prompts, VueAnswer } from './prompts';
+import { Initializer } from '../../common/Initializer';
+import { transform } from '../../common/steps';
+import { VueArgs } from './args';
+import { FetchWith } from '../../common/prompts/base';
+import chalk from 'chalk';
+
+export default class VueInitializer implements Initializer {
+  get isBase() {
+    return true;
+  }
+
+  async init(args: VueArgs) {
+    const defaults = args.yes
+      ? {
+          appName: 'sitecore-jss-vue',
+          hostName: 'https://cm.jss.localhost',
+          fetchWith: FetchWith.GraphQL,
+          appPrefix: true,
+          language: '',
+        }
+      : {};
+
+    const answers = await prompt<VueAnswer>(prompts, { ...defaults, ...args });
+
+    const mergedArgs = {
+      ...args,
+      ...answers,
+    };
+
+    const templatePath = path.resolve(__dirname, '../../templates/vue');
+    await transform(templatePath, mergedArgs, {
+      filter: (filePath) => {
+        return (
+          !filePath.endsWith('index.html') &&
```

Comment:
We need to filter this differently because currently it won't be copied at all. I had the same issue with react-native.
Maybe we can change the delimiters? https://stackoverflow.com/questions/48125577/how-to-change-delimiters-in-vue-js
@illiakovalenko any thoughts on this? (this index.html uses ejs and is causing errors with our ejs runner, so we only want to run it through copy)

Reply:
We don't need to transform it but we need to copy it. It has ejs tokens coming from Vue itself, not us :)